doc/website/blog/2022-09-05-announcing-migration-execution.mdx

```diff
@@ -0,0 +1,390 @@
+---
+title: The Atlas Migration Execution Engine
+authors: masseelch
+tags: [atlas, migrations, versioned, announcement]
+image: https://blog.ariga.io/uploads/images/posts/v0.6.0/atlas-migrate-diff.png
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+With the release of [v0.6.0](https://github.com/ariga/atlas/releases/tag/v0.6.0), we
+[introduced](https://atlasgo.io/blog/2022/08/11/announcing-versioned-migration-authoring) a workflow for managing
+changes to database schemas that we have called: _Versioned Migration Authoring_.
+
+Today, we released the first version of the Atlas migration execution engine, that can apply migration files on
+your database. In this post, we will give a brief overview of the features and what to expect in the future.
+
+### Migration File Format
+
+At Atlas, we believe in a linear migration history. Therefore, Atlas migration files format does not have the concept
+of rollback migrations. To undo a previously applied migration file, simply create a new migration file reverting
+the changes done in the file before.
+
+Now, some might argue, that rollback migration exist to have a fallback mechanism in case a migration execution fails.
+This, however, is deceptive, since
+1. some SQL statements are simply not revertible, (e.g. destructive changes like
+dropping a column),
+2. execution order of the rollback migration is not easy to determine without cluttering the original
+migration and
+3. wrapping migration execution in a transaction makes rollback obsolete, if you are working with a database
+that supports transactional DDL.
+
+Having no support for rollbacks conducts another nice effect: our migration directory consists of a bunch of simple SQL
+files, each of them standing for a version of the database schema. Atlas migration filename format follows a very simple
+structure: `version_[name].sql`, with the `name` being optional. `version` can be an arbitrary string. Migration files
+are lexicographically sorted by filename.
+
+<Tabs
+    defaultValue="tree"
+    values={[
+        {label: 'Tree', value: 'tree'},
+        {label: '1_initial.sql', value: '1'},
+        {label: '2_second.sql', value: '2'},
+        {label: '3_second.sql', value: '3'},
+        {label: 'atlas.sum', value: 'sum'},
+]}>
+<TabItem value="tree">
+
+```shell
+↪ tree .
+.
+├── 1_initial.sql
+├── 2_second.sql
+├── 3_third.sql
+└── atlas.sum
+
+0 directories, 4 files
+```
+
+</TabItem>
+
+<TabItem value="1">
+
+```sql
+-- add new schema named "my_schema"
+CREATE DATABASE `my_schema`;
+-- create "tbl" table
+CREATE TABLE `my_schema`.`tbl` (`col` int NOT NULL);
```

Comment:
I am not sure, if I want to ad more value here like creating `users` and `pets` instead of simple `tbl`s and `col`s.